### ISSUE TITLE
switch from Nokogiri to Ox

### ISSUE THREAD

**Issue by tomasc** [tomasc/dragonfly_svg#2]

**Comment by tvdeyen** [comment]
I don't think this is a good idea. Rails has Nokogiri as dependency, so everybody requiring you're plugin in a rails app would then have two XML parsers in their app.

**Comment by tomasc** [comment]
Well … Nokogiri is a bit difficult because of its use of C library, and I would not rely on Rails having it as a dependency forever. We're using [Ox](https://github.com/ohler55/ox) elsewhere and it's just fine.

**Comment by tvdeyen** [comment]
I know. What about making a check if `Nokogiri` is defined, and require ox only if not.

**Comment by tomasc** [comment]
This is not urgent. We can keep this as is, and only eventually replace Nokogiri with something more lightweight.
